assets/scripts/app/load_resources.js

```diff
@@ -126,23 +86,88 @@ function loadSVGs () {
 }
 
 /**
- * Wraps `new Image()`'s onload and onerror properties with
- * Promises. This provides a more reliable way of knowing when a
- * image element is done loading (rather than fetch) because a
- * fetch can successfully load an image file but be resolved
- * before the image element is ready. So we must resolve only after
- * the image's `onload` callback has been called.
+ * Gets a string representation of an <svg> via its outerHTML property
+ * or with a fallback method if outerHTML is unavailable (IE / Edge)
+ *
+ * @param {SVGElement} svg
+ * @return {string}
+ */
+function getSVGOuterHTML (svg) {
+  let outerHTML = svg.outerHTML
+
+  // If `outerHTML` is not available, use this alternate method below
+  if (typeof outerHTML === 'undefined') {
+    outerHTML = new XMLSerializer().serializeToString(svg)
+  }
+
+  return outerHTML
+}
+
+/**
+ * Gets a string representation of an svg <symbol>'s contents via
+ * the innerHTML property or with a fallback method if innerHTML is
+ * unavailable (IE / Edge)
+ *
+ * @param {SVGSymbolElement}
+ * @return {string}
+ */
+function getSVGSymbolInnerHTML (symbol) {
+  let innerHTML = symbol.innerHTML
+
+  // If `innerHTML` is not available, use this alternate method below,
+  // which iterates through each of the symbol's child nodes and
+  // serializes each element to a string.
+  if (typeof innerHTML === 'undefined') {
+    innerHTML = ''
+    Array.prototype.slice.call(symbol.childNodes).forEach(function (node, index) {
+      innerHTML += (new window.XMLSerializer()).serializeToString(node)
```

Comment:
Do we need `window.` here if we're not using it above?

Reply:
I'll normalize them with `window.`